Contact/ContactRepository.cs

```diff
@@ -42,27 +46,44 @@ public Contact Get(ContactRepository database, int id)
         }
 
         //Delete - set a flag
-        public  void Delete(ContactRepository database, int id)
+        public void Delete(ContactRepository database, int id)
         {
             database.ContactList.Find(i => i.contactID.Equals(id)).isActiveRecord = false;
         }
 
         //Search all fields for matches
-        public  List<Contact> Filter(ContactRepository database, string parameter)
+        public List<Contact> Filter(ContactRepository database, string parameter)
```

Comment:
there is no reason to pass in a ContactRepository into every method of the ContactRepository.